docs/definition_of_done.md

```diff
@@ -0,0 +1,46 @@
+# Definition of Done
+- er is zijn BDD-scenario's (functionele specificatie)
+- er is een Open API specificatie (versie 3)
+- er is beoordeeld op optimale code generatie
+- er is een referentie-implementatie
```

Comment:
Is dit nog steeds relevant?

Reply:
Ik denk dat er in ieder geval een omgeving moet zijn waar de werking van de API te testen is. Dat is nu de omgeving van processFive. Ik denk dat dit dus wel relevant is. 
Als je dat anders ziet, wil je daar dan een issue voor opvoeren. Dan kunnen we die discussie in het team voeren, maar dat houd dan deze PR niet gegijzeld.

Reply:
Ik denk meer dat het team zichzelf gijzelt, want nu kan je niet zeggen dat je 'done' bent totdat er een referentie implementatie is. Volgens mij is de test implementatie van processFive wat anders dan een referentie implementatie. Maar ik vind het prima om het zo te laten

Reply:
Ik heb voor deze discussie een isue opgevoerd.